app/src/components/notifications/PageNotifications.vue

```diff
@@ -19,7 +19,12 @@
           @loadMore="loadMore"
         >
           <template slot-scope="event">
-            <router-link :key="event.id" class="notification" :to="event.link">
+            <router-link
+              :key="event.id"
+              class="notification"
+              :to="event.link.includes('transactions') ? '' : event.link"
+              :class="{ disabled: event.link.includes('transactions') }"
+            >
```

Comment:
can you hide `<i class="material-icons notranslate">chevron_right</i>` for transactions